ospfd/ospf_opaque.c

```diff
@@ -1621,10 +1630,13 @@ ospf_opaque_lsa_install (struct ospf_lsa *lsa, int rt_recalc)
 struct ospf_lsa *
 ospf_opaque_lsa_refresh (struct ospf_lsa *lsa)
 {
-  struct ospf *ospf;
+  struct ospf *ospf = NULL;
   struct ospf_opaque_functab *functab;
   struct ospf_lsa *new = NULL;
   
+  if (IS_DEBUG_OSPF_TRACE)
+    zlog_debug ("%s: default ospf lookup. opaque lsa->oi->ospf->vrf_id %u",
+                __PRETTY_FUNCTION__, lsa->oi->ospf->vrf_id);
   ospf = ospf_lookup ();
```

Comment:
leaving two api's around that do the same thing, one vrf aware and one not is a severe recipe for bugs.  I've already fixed multiple instances of bugs introduced by having a non vrf aware api called when it should have been using the vrf aware.  Just remove ospf_lookup and replace with an appropriate vrf aware function